build-server/main.sh

```
@@ -0,0 +1,23 @@
+ENV_VARS="${ENV}"
+
+GIT_REPOSITORY_URL="https://github.com/${USERNAME}/${REPO}.git"
+
+
+git clone "$GIT_REPOSITORY_URL" 
+cd "$REPO"
+git checkout "$BRANCH"
+
+if [ -n "$ENV_VARS" ]; then
+  echo "$ENV_VARS" | jq -r '.[] | "\(.key)=\(.value)"' > ".env"
+  export $(grep -v '^#' ".env" | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)